src/modules/discovery/kube_state_metrics.py

```diff
@@ -0,0 +1,42 @@
+
+import logging
+import subprocess 
+import json
+
+from ...core.types import Discovery
+from ...core.events import handler
+from ...core.events.types import HuntStarted, Event
+
+
+class KubeStateMetricsEvent(Event):
+    """Add-on agent to generate and expose cluster-level metrics."""
+    def __init__(self, version):
+        self.version = version
+
+# Will be triggered on start of every hunt 
+@handler.subscribe(HuntStarted)
+class KubeStateMetricsDiscovery(Discovery):
+    """kube-state-metrics version Discovery
+    Checks the version of kube-state-metrics part of CVE-2019-17110 vulnerability check
```

Comment:
could you please remove the reference to the CVE? the discovered service might be useful for other hunters in the future

Reply:
Ok.